src/main/java/org/opensearch/security/auditlog/config/AuditConfig.java

```diff
@@ -130,9 +131,9 @@ public static AuditConfig from(final Settings settings) {
      */
     @JsonInclude(JsonInclude.Include.NON_NULL)
     public static class Filter {
+        private static Set<String> FIELDS = DefaultObjectMapper.getFields(Filter.class);
```

Comment:
After adding changes to the `.from` method, this made the order of these static fields was incorrect causing static constructors to fail.  A great reminder to use lazy loading avoiding static order of operations management.